src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -741,7 +738,8 @@ static void create_rr_locked(grpc_exec_ctx *exec_ctx, glb_lb_policy *glb_policy,
   rr_connectivity->state = rr_state;
 
   /* Subscribe to changes to the connectivity of the new RR */
-  GRPC_LB_POLICY_WEAK_REF(&glb_policy->base, "rr_connectivity_sched");
+  GRPC_LB_POLICY_WEAK_REF(&glb_policy->base, "glb_rr_connectivity_sched");
+  GRPC_LB_POLICY_WEAK_REF(glb_policy->rr_policy, "rr_connectivity_sched");
```

Comment:
It's not, removed. 

Testing showed it was needed before I refactored this code to process `shutdown` separately (the real issue). My reasoning was that, given that the callback references the RR instance as well, a weak ref ought to be kept. But of course, the grpclb instance owns and is responsible for the RR instance already.